dll/cpl/sysdm/lang/fr-FR.rc

```diff
@@ -325,6 +325,10 @@ END
 
 STRINGTABLE
 BEGIN
+    IDS_EDITVARCAPTION "Edit "
+    IDS_NEWVARCAPTION "New "
+    IDS_USERVAR "user variable"
+    IDS_SYSTEMVAR "system variable"
```

Comment:
```suggestion
    IDS_EDITVARCAPTION "Modifier la "
    IDS_NEWVARCAPTION "Nouvelle "
    IDS_USERVAR "variable utilisateur"
    IDS_SYSTEMVAR "variable système"
```

Reply:
You have to open a new PR for the French translation

Reply:
Why the Portuguese translation then ?